src/Maker/MakeDockerDatabase.php

```diff
@@ -80,13 +80,22 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
     {
         $io->section('- Docker Compose Setup-');
 
+        $composeFileExtensions = ['.yml', '.yaml'];
+        $composeFileExists = false;
         $composeFileContents = '';
         $statusMessage = 'Existing docker-compose.yaml not found: a new one will be generated!';
 
-        if ($this->fileManager->fileExists($this->composeFilePath)) {
-            $composeFileContents = $this->fileManager->getFileContents($this->composeFilePath);
```

Comment:
It looks like we currently calculate the `$this->composeFilePath` property in the constructor. Instead of doing that logic (and instead of having the inline logic here), could we remove that property and replace it with a `private function getComposeFilePath(): ?string`? That would loop over the possible paths and return the file path if it was found. Then, most of this section could stay the same, except that we would call the method instead of referencing the property.

Reply:
We need access to that property in `generate()` to be able to dump the new / update content. The latest commit adds a `getComposeFileContents()` that determines the file && grabs its contents if it exists. I think this is as close as we can get before we start abusing input args. 

We can revert the logic split back to inline if this implementation isn't as appetizing.